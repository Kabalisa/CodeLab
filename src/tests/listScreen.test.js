import ListScreen from '../screens/listScreen';

describe('ListScreen tests', () => {
  const props = { navigation: { navigate: jest.fn() } };
  it('should render without error', () => {
    const wrapper = shallow(<ListScreen {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
