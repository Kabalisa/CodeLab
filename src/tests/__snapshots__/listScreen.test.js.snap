// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListScreen tests should render without error 1`] = `
<View>
  <Header
    navigation={
      Object {
        "navigate": [MockFunction],
      }
    }
  />
  <View
    style={
      Object {
        "flexDirection": "column",
        "height": 695,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 18,
          "fontWeight": "bold",
          "left": 15,
          "top": 20,
        }
      }
    >
      Java Developers In Lagos:
    </Text>
    <View
      style={
        Object {
          "height": 600,
          "marginHorizontal": 30,
          "marginTop": 50,
        }
      }
    >
      <QueryComponent
        navigation={
          Object {
            "navigate": [MockFunction],
          }
        }
      />
    </View>
  </View>
  <Footer />
</View>
`;
